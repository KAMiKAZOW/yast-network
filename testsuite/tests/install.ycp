{

// testedfiles: install.ycp

include "testsuite.ycp";

map READ = $[
    "etc" : $[ "install_inf" : $[ "NetConfig" : "bootp" ]],
];

map InstallInf = $[];
include "network/install.ycp";

// DUMP("ReadInstallInf");
// TEST(``(ReadInstallInf()), [], nil);

InstallInf = $[];
path install_inf = .etc.install_inf;

DUMP("ReadInstallInfItem");
TEST(``(ReadInstallInfItem(install_inf, "bootproto", "NetConfig")), [READ], nil);
DUMP(InstallInf);
TEST(``(ReadInstallInfItem(install_inf, "ipaddr", "IP")), [READ], nil);
DUMP(InstallInf);

}
