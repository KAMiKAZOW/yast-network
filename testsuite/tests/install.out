Dump	ReadInstallInfItem
Read	.etc.install_inf."NetConfig" "bootp"
Return	true
Dump	$["bootproto":"bootp"]
Read	.etc.install_inf."IP" nil
Return	false
Dump	$["bootproto":"bootp"]
